Mathlib/NumberTheory/DirichletCharacter/Basic.lean

```diff
@@ -199,4 +209,21 @@ lemma conductor_one_dvd (n : ℕ) : conductor (1 : DirichletCharacter R 1) ∣ n
   rw [(isPrimitive_def _).mp isPrimitive_one_level_one]
   apply one_dvd _
 
+/-- The primitive character associated to a Dirichlet character. -/
+noncomputable def reduction : DirichletCharacter R χ.conductor :=
```

Comment:
I think "reduction" is too generic. Why not `primitiveCharacter` instead?

Reply:
`primitiveCharacter` is quite long, I am happy with `PrimitiveChar` or simply `Primitive`